Modules/HR/Http/Controllers/Universities/ReportController.php

```diff
@@ -16,4 +17,33 @@ public function index()
 
         return view('hr::universities.reports');
     }
+
+    public function show(University $report)
+    {
+        $datas = DB::table('hr_applications')
```

Comment:
_Data_ is a plural word. The singular is _datum_.
```suggestion
        $data = DB::table('hr_applications')
```